tests/frontendIntegration/django3x/polls/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.urls import path
 
-from tests.frontendIntegration.django3x.polls import views
+from . import views
```

Comment:
`from tests.frontendIntegration.django3x.polls import views` always blocked me from running pre-commit hooks. Pyright throws errors (`reportMissingTypeStubs`). That's why I often used `--no-verify`. 

Fixing that problem today. No more `--no-verify` from my side :)

Reply:
ok